napari/layers/_labels_layer/model.py

```diff
@@ -107,6 +147,130 @@ def data(self, data):
         self._image, self._meta = data
         self.refresh()
 
+    @property
+    def contiguous(self):
+        """ bool: if True, fill changes only pixels of the same label that are
+        contiguous with the one clicked on.
+        """
+        return self._contiguous
+
+    @contiguous.setter
+    def contiguous(self, contiguous):
+        self._contiguous = contiguous
+        self.events.contiguous()
+
+        self.refresh()
+
+    @property
+    def n_dimensional(self):
+        """ bool: if True, edits labels not just in central plane but also
+        in all n dimensions according to specified brush size or fill.
+        """
+        return self._n_dimensional
+
+    @n_dimensional.setter
+    def n_dimensional(self, n_dimensional):
+        self._n_dimensional = n_dimensional
+        self.events.n_dimensional()
```

Comment:
Side comment: I wonder whether n-dimensional is ever something we want...

Reply:
agreed - it may not be so important. i'm curious for the people doing volumetric stuff. I'd say we leave it in for now and can always remove if unwanted or hard to support